packages/earljs/src/validators/other/toThrow.ts

```diff
@@ -54,7 +53,8 @@ declare module '../../expect' {
      * `toBeRejectedWith` instead.
      *
      * @param errorClass - The error class to check.
-     * @param message - A substring of the error message or a regex matching the message.
+     * @param message - A substring of the error message or a regex matching the
+     *   message.
```

Comment:
See if VSCode renders this differently than the same thing on a single line.

Reply:
Yeah it does but I am not sure why. tsdoc docs says it should be fine. https://tsdoc.org/pages/tags/param/